main/rele.cpp

```diff
@@ -0,0 +1,43 @@
+#include "rele.h"
+
+
+Rele::Rele() {
+    // Defina o número do pino apropriado para o relé
+    m_rele = GPIO_NUM_8;
+    // Configura o pino do relé como saída
+    gpio_pad_select_gpio(m_rele);
+    gpio_set_direction(m_rele, GPIO_MODE_OUTPUT);
+    gpio_set_level(m_rele, 0); // Desliga o relé inicialmente
+}
+
+void Rele::on() {
+    gpio_set_level(m_rele, 1); // Liga o relé
+}
+
+void Rele::off() {
+    gpio_set_level(m_rele, 0); // Desliga o relé
+}
+
+void Rele::toggle() {
+    int level = gpio_get_level(m_rele);
+    gpio_set_level(m_rele, !level); // Inverte o estado do relé
```

Comment:
```suggestion
    gpio_set_level(m_rele, !level); // Inverte o estado do relé
```

Revê esse pedaço. Acho que não tem como ler num pino que foi definido como output